package.json

```diff
@@ -7,7 +7,7 @@
   ],
   "scripts": {
     "build": "lerna run build",
-    "build:clean": "yarn clean:prisma && rimraf \"packages/**/dist\"",
+    "build:clean": "yarn clean:prisma && rimraf \"packages/**/dist\" --glob",
```

Comment:
new config required. Note: I tested locally to confirm working, especially in case of test directories